style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  font-family: sans-serif;
+  padding-left: 20%;
+  padding-right: 20%;
+}
+
+label {
+  font-size: 18px;
+  display: block;
+  padding-bottom: 10px;
+}
+
+input,
+textarea,
+select {
+  display: block;
+  width: 100%;
+  padding-bottom: 20px;
+}
```

Comment:
I think it would be better to use class attributes when you want to implement the same style to the different parts of your code instead of using the name of the tags 👍 😃